app/admin/todo_list.rb

```diff
@@ -4,7 +4,8 @@
   # See permitted parameters documentation:
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   #
-  permit_params :title, :deadline, todo_items_attributes: [:title, :description]
+  permit_params :title, :deadline,
+                todo_items_attributes: [:id, :title, :description, _destroy: true]
```

Comment:
Jeez HoundCI

-- 
Anonoz Chong
Sent with Airmail

On March 8, 2015 at 4:10:08 AM, Hound (notifications@github.com) wrote:

In app/admin/todo_list.rb:

> @@ -4,7 +4,8 @@
>    # See permitted parameters documentation:
>    # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
>    #
> -  permit_params :title, :deadline, todo_items_attributes: [:title, :description]
> +  permit_params :title, :deadline,
> +                todo_items_attributes: [:id, :title, :description, _destroy: true]
Line is too long. [82/80]

—
Reply to this email directly or view it on GitHub.